src/app/components/Tab.tsx

```diff
@@ -34,7 +46,7 @@ type Vnode = m.Vnode<Attrs, Tabs>;
 export default class Tabs extends MithrilTsxComponent<Attrs> {
```

Comment:
The file is called Tab and the class is called `Tabs`. Better to make them both the same.